aten/src/ATen/native/LinearAlgebraUtils.h

```diff
@@ -48,22 +48,33 @@ static inline int64_t matrixStride(const Tensor& batched_matrices) {
  * Here batch contiguity is checked for tensors with greater than 4 dimensions.
  * Contiguous tensors and tensors with less than 3 dimensions pass this check
  */ 
-static inline bool checkTrilTriuBatchContiguous(const Tensor& tensor) {
+static inline std::tuple<bool, Tensor> checkTrilTriuBatchContiguous(const Tensor& tensor) {
```

Comment:
nit: Returning two arguments like this is confusing to read (at least for me), but I'm not sure how to improve it.

Maybe a signature like
```
Tensor coerceTrilTriuBatchContiguous(const Tensor& tensor, bool* was_already_batch_contiguous)
```
might be clearer.

Reply:
I thought about keeping the code more descriptive, and hence didn't consider a LAPACK-like return strategy.

But if you think that it reads better, I can make a change like below:
```c++
Tensor coerceTrilTriuBatchContiguous(const Tensor& tensor, bool& was_already_batched) {
  ...
}
```

Reply:
Let's just keep what you have for now

Reply:
Sure, thanks for the review.